java/examples/pom.xml

```diff
@@ -50,10 +50,12 @@
     <dependency>
       <groupId>org.apache.hadoop</groupId>
       <artifactId>hadoop-common</artifactId>
+      <scope>compile</scope>
     </dependency>
     <dependency>
       <groupId>org.apache.hadoop</groupId>
       <artifactId>hadoop-hdfs</artifactId>
+      <scope>compile</scope>
```

Comment:
BTW, @krystalics . When you make a PR, you had better rebase your branch to the latest `main` branch.
For example, this file is the content of the exiting `main` branch, but it seems that GitHub shows this because your PR branch is too old.

Reply:
i fork it at yesterday, it changes so fast